akka-sample-camel-java/README.md

```diff
@@ -1,12 +1,12 @@
-This tutorial contains 3 samples of [Akka Camel](http://doc.akka.io/docs/akka/2.5.0/java/camel.html).
+This tutorial contains 3 samples of [Akka Camel](http://doc.akka.io/docs/akka/current/java/camel.html).
```

Comment:
Wouldn't the 2.5 link be better? (Minor thing, but this will point to 2.6 when we release it even if we haven't updated the samples yet) Using http://doc.akka.io/docs/akka/2.5/java/camel.html would still save us updating on every release.